catalog/networking/network/README.md

```diff
@@ -1,28 +1,18 @@
 # Network blueprint
 
-This package creates a network with VPN and NAT configuration.
+A private network with VPN and Cloud NAT.
 
-Contents:
+**Requires a pre-existing secret!**
 
--   network.yaml - contains network configuration.
--   nat.yaml - contains cloud NAT configuraiton.
--   vpn.yaml - contains cloud VPN configuration.
-
-## Applying the blueprint
-
-NOTE: Replace all environment variables in the example below with the
-appropriate values.
-
-Before applying this blueprint you need to create a Kubernetes secret object
-with the VPN key.
-
-This could be done either via a command line:
+Via kubectl:
 
 ```shell
-kubectl create secret generic vpn-shared-secret --from-literal=vpn-shared-secret=${VPN_SECRET_VALUE} -n ${NETWORK_NAMESPACE}
+kubectl create secret generic vpn-shared-secret \
+    --from-literal "vpn-shared-secret=${VPN_SECRET_VALUE}" \
+    -n ${NETWORK_NAMESPACE}
 ```
 
-Or declaratively:
+Or Config Connector:
```

Comment:
Not sure what this has to do with Config Connector?

Let's stick to declaratively (since this can just be done with `kubectl apply`).

Reply:
fixed